### PR TITLE
Don't read from environment when auto-resolving

### PR DESCRIPTION
**Pull request by maroux**
Fixes GoogleCloudPlatform/berglas#45

### CODE REVIEW

File: pkg/berglas/resolver.go [hunk 1]
```diff
@@ -86,11 +86,10 @@ func Replace(ctx context.Context, key string) error {
...
-// Replace parses a berglas reference from the environment variable at the
-// given environment variable name. If parsing and extraction is successful,
-// this function replaces the value of the environment variable to the resolved
-// secret reference.
-func (c *Client) Replace(ctx context.Context, key string) error {
+// ReplaceValue parses a berglas reference from value. If parsing and extraction
+// is successful, this function sets the value of the environment variable to the
+// resolved secret reference.
+func (c *Client) ReplaceValue(ctx context.Context, key string, value string) error {
 	plaintext, err := c.Resolve(ctx, os.Getenv(key))
```
Comment (tmthrgd) [inline pkg/berglas/resolver.go]: This PR neglected to actually use `value` here. This call to `os.Getenv(key)` should have been `value`.

Comment (maroux) [reply pkg/berglas/resolver.go]: Ah damnit you're right.

Comment (maroux) [reply pkg/berglas/resolver.go]: Opened new PR #53

**Pull request merged**